aodntools/timeseries_products/gridded_timeseries.py

```diff
@@ -209,7 +209,7 @@ def grid_variable(file_name, VoI, depth_bins=None, max_separation=50, depth_bins
     ## create the output file name and write the aggregated product as netCDF
     facility_code = TStools.get_facility_code(file_name)
     data_code = TStools.get_data_code(VoI) + 'Z'
-    product_type = 'gridded-timeseries'
+    product_type = 'timeSeriesProfile'
```

Comment:
Sorry, this is not what I meant in my email. 'gridded-timeseries' is fine in the file name.
It's the `feature_type` global attribute that should be 'timeSeriesProfile' (or removed, which might be necessary to pass the compliance checks)

Reply:
fixed. It is featureType without the underscore, coming from the original files and passed to the hourly. It is quite weird as all the other two-word global attrs are underscore separated.

Reply:
Yes, it's inconsistent, but `featureType` is what CF expects (http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#featureType)